NAMESPACE

```diff
@@ -18,6 +18,17 @@ export(xportr_write)
 export(xpt_validate)
 import(haven)
 import(rlang)
+importFrom(checkmate,assert)
+importFrom(checkmate,assert_character)
+importFrom(checkmate,assert_choice)
+importFrom(checkmate,assert_data_frame)
+importFrom(checkmate,assert_integer)
+importFrom(checkmate,assert_logical)
+importFrom(checkmate,assert_string)
+importFrom(checkmate,check_data_frame)
+importFrom(checkmate,check_r6)
+importFrom(checkmate,test_r6)
+importFrom(checkmate,test_string)
```

Comment:
Maybe we can use `checkmate::assert_` and just `import(checkmate)` here? It will be nice to no have access to `assert_` functions after loading `xportr` WDYT?

Reply:
My bad, they don't get attached while loading the package using `library`